python/GafferUI/NodeSetEditor.py

```diff
@@ -327,11 +327,6 @@ def acquire( cls, node, floating = None ) :
 			window = _EditorWindow( scriptWindow, editor )
 			# Ensure keyboard shortcuts are relayed to the main menu bar
 			scriptWindow.menuBar().addShortcutTarget( window )
-			## \todo Can we do better using `window.resizeToFitChild()`?
-			# Our problem is that some NodeEditors (for GafferImage.Text for instance)
-			# are very large, whereas some (GafferScene.Shader) don't have
-			# a valid size until the UI has been built lazily.
-			window._qtWidget().resize( 400, 400 )
```

Comment:
The new behaviour is stella for Spreadsheets, sadly I my innocent request may be a confounded request, I think it's a little bit of a backwards step for many common nodes that rely on expandable plug layouts. Eg: this for a `StandardOptions` node:

![unnamed](https://user-images.githubusercontent.com/896779/77766046-09e90600-7037-11ea-9081-40cdbf8ea825.png)

It creates quite a bit of faf for the user having to resize the window once they have expanded a section. Could we perhaps still enforce 400 x 400 as a minimum size here?

Reply:
acdf66f. Kinda sucks, because many things now open bigger than needed and require manual adjustment to avoid space wastage. But it's not worse than before in that respect, and we have better widths now...

Reply:
Thanks John, think thats a good compromise.